### DISCUSSION TITLE
Security Onion Sensors

### DISCUSSION THREAD

**Discussion by larstchet** [Security-Onion-Solutions/securityonion#8010] [Unsupported Versions] [answered]
Is there a way to be alerted when one of my sensor interfaces goes down? I would like to receive an email? My version of SO is 2.3.

Thanks, 

Larry Stover

**Comment by dougburks** [comment, marked as answer]
Take a look at our Grafana video and documentation:
https://www.youtube.com/watch?v=8FmZ4MRe8Uk
https://docs.securityonion.net/en/2.3/grafana.html#example
